src/main/scala/cognite/spark/v1/PushdownUtilities.scala

```diff
@@ -119,11 +119,11 @@ object PushdownUtilities {
       case _ => {
         val minimumTimeAsInstant =
           minTime
-            .map(java.sql.Timestamp.valueOf(_).toInstant.plusMillis(1))
+            .map(java.sql.Timestamp.valueOf(_).toInstant)
```

Comment:
We probably had a reason for the old `plusMillis`, any idea why we did that? :/

Reply:
I _think_ that's because we may initially have coded this without pushdowns for `gt/ltOrEqual`. As far as I can see the endpoints are inclusive, so this is the right way to always get the datapoints (and filter later if < or >).